### PR TITLE
MF-128: Conditions Level Three Widget

### PR DESCRIPTION
**Pull request by denniskigen**
Screenshot 2020-01-27 at 16 03 58

figma link https://www.figma.com/file/YyVhWTNp89bNiOGadYrHBk/Patient-Chart-Master-current-file?node-id=6427%3A189644

### CODE REVIEW

File: src/summaries/level-two-routes.component.tsx [hunk 1]
```diff
@@ -56,6 +57,11 @@ export const levelTwoRoutes: PatientChartRoute[] = [
     url: "/patient/:patientUuid/chart/conditions",
     component: ConditionsBriefSummary,
     name: "Conditions"
+  },
+  {
+    url: "/patient/:patientUuid/chart/conditions/:conditionUuid",
+    component: ConditionsDetailedSummary,
+    name: "Condition"
```
Comment (jonathandick) [inline src/summaries/level-two-routes.component.tsx]: I think this should be "Conditions"

File: src/widgets/cards/summary-card.css [hunk 2]
```diff
@@ -16,10 +16,19 @@
...
   color: var(--omrs-color-interaction-plus-one);
 }
 
+button.editBtn:hover {
+  color: var(--omrs-color-interaction-plus-one);
+}
+
 .header {
   display: grid;
   grid-template-columns: [one] 1fr [two] 68px;
```
Comment (jonathandick) [inline src/widgets/cards/summary-card.css]: Please don't use px, use rem.

File: src/widgets/conditions/conditions-detailed-summary.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  display: flex;
+}
+
+:global(.omrs-breakpoint-lt-tablet) .conditionSummary {
+  padding: 0.25rem 0.5rem 0 0.5rem;
+}
+
+.conditionCard {
+  margin: 0rem 1rem;
+  font-size: 1.063rem;
```
Comment (jonathandick) [inline src/widgets/conditions/conditions-detailed-summary.css]: Why such an unusual font size?

File: src/summaries/level-two-routes.component.tsx (hunk 1 above)
Comment (denniskigen) [reply src/summaries/level-two-routes.component.tsx]: This route is for the detailed view when you click on an individual condition - so the singular form should be appropriate for it.

File: src/widgets/cards/summary-card.css (hunk 2 above)
Comment (denniskigen) [reply src/widgets/cards/summary-card.css]: Ok. I'll change this.

File: src/widgets/conditions/conditions-detailed-summary.css (hunk 3 above)
Comment (denniskigen) [reply src/widgets/conditions/conditions-detailed-summary.css]: 17px in keeping with the design, though 16px (which neatly converts to 1rem) would probably be just as appropriate (and makes for a cleaner conversion). Should I go with 17px as per the design?

Comment (jonathandick) [reply src/widgets/conditions/conditions-detailed-summary.css]: Please stick with design.

File: src/summaries/level-two-routes.component.tsx (hunk 1 above)
Comment (jonathandick) [reply src/summaries/level-two-routes.component.tsx]: In "Chart -> Conditions -> Renal Failure", the "Conditions" refers to the detailedSummary widget, so the link is to "Conditions". The choice of name within a breadcrumb is independent of the final node in the breadcrumb

File: src/widgets/conditions/conditions-detailed-summary.css (hunk 3 above)
Comment (denniskigen) [reply src/widgets/conditions/conditions-detailed-summary.css]: Ok. 1.063 rem it is.

**Pull request merged**